pick.lic

```diff
@@ -354,6 +354,24 @@ class LockPicker
     end
     waitrt?
     case bput("pick my #{box} #{speed}", 'You discover another lock protecting', 'You are unable to make any progress towards opening the lock', 'Roundtime', /Find a more appropriate tool and try again/)
+    when 'You are unable to make any progress towards opening the lock'
```

Comment:
Looking at the code in this method, this seems really verbose for a single case - maybe want to extract this logic to a method?